packages/common/lib/types/CredentialIssuance.types.ts

```diff
@@ -12,8 +12,11 @@ export interface CredentialRequest {
 }
 
 export interface CredentialResponse {
-  credential: W3CVerifiableCredential;
-  format: CredentialFormat | CredentialFormat[];
+  credential?: W3CVerifiableCredential; // OPTIONAL. Contains issued Credential. MUST be present when acceptance_token is not returned. MAY be a JSON string or a JSON object, depending on the Credential format. See Appendix E for the Credential format specific encoding requirements
```

Comment:
Since you now made credential optional, I would expect an expression somewhere checking the value to be present before V11, as it was required back than. Don't just change optionality without actually doing something with it if it changed between versions

Reply:
Acceptance_token is part of the deferred response, so for now I will only validate that the credential is actually there (mandatory since deferred response is not implemented).